### PR TITLE
Add link to GL library

### LINKED ISSUE #109: Revive link to GL library.
The link to GL library was removed by #106, since gleam does not call linked GL functions. But it is necessary to revive it. Dependent crates expects that gleam does link to GL library to avoid double linking error like emilio/rust-offscreen-rendering-context#84.

### PR DESCRIPTION
**Pull request by sotaroikeda**
This fixes #109.

---
This change is [Reviewable](https://reviewable.io/reviews/servo/gleam/110)

### CODE REVIEW

**Comment by sotaroikeda** [comment]
@emilio, can you review it?

**Comment by sotaroikeda** [comment]
@emilio, which is better to add GL link to gleam or offscreen_gl_context.

File: Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "gleam"
-version = "0.3.0"
+version = "0.3.1"
```
Comment (emilio) [inline Cargo.toml]: I believe this is a breaking change, and @nox will punish me if I r+ it in this state.

**Comment by sotaroikeda** [comment]
Thanks for the comment! I am going to defer linking to offscreen_gl_context.

**Pull request closed without merging**